src/components/Recoil_RecoilRoot.react.js

```diff
@@ -213,7 +213,7 @@ function RecoilRoot({initializeState, children}: Props): ReactElement {
       return;
     }
     // Save changes to nextTree and schedule a React update:
-    if (__DEV__) {
+    if (__DEV__ && typeof window !== 'undefined') {
```

Comment:
Can we use a standard `window != null` check?

Reply:
I believe `typeof` is the only way to check if the window object is valid on the server since `window` is never declared, checking if it's `null` would still error out